projects/kubernetes-sigs/image-builder/patches/0013-Add-non-efi-builds-for-THEL-9-ova.patch

```diff
@@ -204,5 +201,5 @@ index 0b806d1a8..91e351f1a 100644
    "http_directory": "./packer/ova/linux/{{user `distro_name`}}/http/",
    "iso_checksum": "398561d7b66f1a4bf23664f4aa8f2cfbb3641aa2f01a320068e86bd1fc0e9076",
 -- 
-2.49.0
+2.48.1
```

Comment:
We usually keep this value consistent for all patches, so we probably don't want to change this